src/style.css

```
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts\Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts\Avenir-Heavy.ttf);
+}
```

here is a better approach: font-family is the same, we just indicate font-weight (so that we know what font to use by indicating its font-weight)
```suggestion
@font-face {
  font-family: Avenir;
  src: url(fonts\Avenir-Book.ttf);
  font-weight: normal;
}

@font-face {
  font-family: Avenir;
  src: url(fonts\Avenir-Heavy.ttf);
  font-weight: bold;
}
```

---

I got it, will implement it to my code. Should i push it once again after implementing changes?

---

yes, let's commit changes, push them, and then re-run deploy

---

Done